presto/presto.go

```diff
@@ -1366,67 +1351,7 @@ func getScanType(typeNames []string) (reflect.Type, error) {
 		v = sql.NullInt64{}
 	case "real", "double":
 		v = sql.NullFloat64{}
-	case "date", "time", "time with time zone", "timestamp", "timestamp with time zone":
-		v = sql.NullTime{}
```

Comment:
Updated date and time types to be returned as strings instead of as `time.Time`, since we usually want to show column values to end users in the same format in which they're returned from the underlying database.